### ISSUE TITLE
1-Wire thermometer return 0 once in a while

### ISSUE THREAD

**Issue by somm15** [JoDehli/PyLoxone#36]
Hi !

Thanks again for the integration. It works very well. 

I just installed 1-Wire thermometer and overall it works, sensor are ok in HA.
However, for a reason, every 10 to 20 minutes, I get the value 0 for all thermometer in HA. 
But I monitor the 1-wire bus on Loxone and no 0 value is reported. 

I have one hypothesis but I cannot confirm yet. It looks like if the temperature does not change, there is no notification on the web socket. I tried to change the "minimal change" value to increase notification frequency. I'll see if it improves something.

**Comment by somm15** [comment]
The hypothesis looks promising. I changed the "minimal" change from 0.25 to 0.1 and it appears to work. Unfortunately, I don't know enough about the internals of HA. 
I'm running event monitor in // with 1-wire monitor and temperature graph, it looks like HA receive events only on value change. I guess if the temperature is too stable it can cause problems.

**Comment by JoDehli** [comment]
Could be a rounding problem. Do they show up as analog sensors in HA? You can improve the digits in the format in the loxone and try it again.

**Comment by somm15** [comment]
They do show up as analog sensor. 
I spent quite a bit of time debugging yesterday. Loxone does not have any problems with the sensors. Values are correct. 
I think I soled it by stopping the Loxone integration, deleting all entities and re-enabling Loxone. 
I had almost no error during the night so I suspend it could be a conflict with another entities (e.g. motion sensor). 
I'll monitor for a few days and close the issue if appropriate.
